docs/reproducibility.md

```diff
@@ -0,0 +1,44 @@
+# Reproducibility
+
+The assumption here is that the original build was done using a Docker build. The
+goal is to reproduce the same MRENCLAVE value. 
+
+You need a couple pieces of information to begin:
+
+- The source MRENCLAVE value, get it from `https://ledgerurl/node/quotes/self`, e.g.:
+
+    ```json
+    {
+        "endorsements": "AQAAAAIAAADZL...UMTI6NTU6MTFaAA==",
+        "format": "OE_SGX_v1",
+        "mrenclave": "fb2c496416fbab20837fedda0ba6db58d819fa5f5c1b3916062eb2fb9d889966",
+        "node_id": "247f1df23e22256cc5bc5e8822183117bc5967da41a257d307f9b1153a4f1853",
+        "raw": "AwACAAAAAAAIAA...ViynsClboLw="
+    }
+    ```
+- Git commit id that built this version. This is something that has no specifc mapping at the moment. You would need to check the build logs of the SGX Docker image to understand which commit produced the candidate value. e.g., `fb2c496416fbab20837fedda0ba6db58d819fa5f5c1b3916062eb2fb9d889966` was built from `fd77c0c69ee890bdc2fcf6ef0c9dddb7b211e164`.
+
+To reproduce the same MRENCLAVE value which would be deployed to CCF 
+do a docker build locally but inside of the development version of CCF image:
+
+- Clone the repository and check out the tag or commit id that built the binary which had specific MRENCLAVE.
+- Identify the expected CCF version by inspecting `./docker/enclave.Dockerfile`.
```

Comment:
It is possible to get the CCF version also by checking the `/node/version` endpoint on a live instance. Do you think it's worth specifying this here as an alternative?

Reply:
I think no because it is not a guarantee that it was used to build the image